.pipelines/windows-vpack-build.yaml

```diff
@@ -0,0 +1,34 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)_e2e
+trigger: none
+
+pool:
+  name: $(AZURE_POOL_NAME)
+
+jobs:
+- job: Build_Windows_VPack
+  steps:
+    - bash: |
+        az login --identity
+        az account set -s $(AZURE_E2E_SUBSCRIPTION_ID)
+        
+        VPACK_VM_RESOURCE_GROUP_NAME="vpack-build-$(date +%s)-$RANDOM"
+        echo "##vso[task.setvariable variable=VPACK_VM_RESOURCE_GROUP_NAME]$VPACK_VM_RESOURCE_GROUP_NAME"
+        IMAGE_REFERENCE="MicrosoftWindowsServer:WindowsServer:2022-datacenter-azure-edition-core:latest"
+        # IMAGE_REFERENCE="/subscriptions/$AZURE_E2E_IMAGE_SUBSCRIPTION_ID/resourceGroups/$AZURE_E2E_IMAGE_RESOURCE_GROUP_NAME/providers/Microsoft.Compute/galleries/$AZURE_E2E_IMAGE_GALLERY_NAME/images/windows-$WINDOWS_OS_VERSION/versions/latest"
+
+        az group create -g $VPACK_VM_RESOURCE_GROUP_NAME -l westus
+        az vm create -n vpack-build-vm -g $VPACK_VM_RESOURCE_GROUP_NAME --image $IMAGE_REFERENCE --assign-identity $(AZURE_MSI_RESOURCE_STRING) --admin-password "!Ab0123456789"
+      displayName: Create Windows VM
+    
+    - bash: |
+        cd e2e/windows/
+        az vm run-command invoke --command-id RunPowerShellScript \
+          --resource-group $VPACK_VM_RESOURCE_GROUP_NAME \
+          --name vpack-build-vm \
+          --scripts @vpack.ps1 \
+          --parameters arg1=$(GitBranch) arg2=$(VpackName) arg3=$(VpackFolder) arg4=$(UserName) arg5=$(PAT)
+      displayName: Exectue VPack Script
+    
+    - bash: |
+        az group delete --name $VPACK_VM_RESOURCE_GROUP_NAME --yes
+      displayName: Delete Windows VM
```

Comment:
can we always run this step to cleanup the resource?

Reply:
Yes, use stage instead of bash step now.